__mocks__/fs.ts

```diff
@@ -1,6 +0,0 @@
-const fs = require.requireActual('fs');
-
-module.exports = {
-  ...fs,
-  writeFile: jest.fn((source: unknown, opts: unknown, cb: Function) => cb(null)),
```

Comment:
any reason to remove the mock? I don't see `fs.writeFile` mocked anymore and we do make such an operation in CLI related tests (output)

Reply:
it was a mistake